pycqed/tests/openql/test_clifford_rb_oql_CC.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
Better to merge this file with the regular RB oql.py . The test is testing the same file just using a different config.

Reply:
Performed this change, and also for the other OpenQL tests